scripts/helpers/buildListingDetails.js

```diff
@@ -82,14 +82,26 @@ export const buildListingDetails = (metadata) => {
 
     if (!builtDetail) return;
 
+    // All details elements live inside the grid container except for locations, which is beside it.
     if (detail.label === "Location") {
       detailsSection.append(builtDetail);
     } else {
       detailGridContainer.append(builtDetail);
     };
   });
 
-  if (detailGridContainer?.children.length > 1)
+  if (detailGridContainer?.children.length > 1) {
     detailsSection.append(detailGridContainer);
+  }
+
+  // Apply now button: duplicate button, where both are displayed at different breakpoints,
+  // so the source order reflects the visuals as a screen reader consideration.
+  if (applyButton !== null) {
+    const appliedButtonOrderedLast = applyButton.cloneNode(true);
+    appliedButtonOrderedLast.classList.add('listing-details__button--last');
```

Comment:
"applied" is a little confusing to me here, is that "apply?" I like the "ordered Last"

Reply:
Yep! I've renamed it.